sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -17,8 +17,8 @@ namespace Azure.Messaging.EventHubs.Tests
     /// </summary>
```

Comment:
Can we add a test for `Consumer can receive events when the client is closed?`     In theory, we should be able to close the `EventHubClient` after creating the consumer and it should still be able to function just fine.

Reply:
Just added a test. Are you sure that's the expected behavior? Both Track One and current Track Two SDKs throw an `ObjectDisposedException`.

Reply:
Well, I had assumed it was...   if Track One goes boom, then lets not do it for now.   We can revisit when we get down into the lower levels for future previews and GA.

Reply:
Looking over the code, I'm pretty sure that the exception is expected base on the architectural decisions;  the `EventHubClient` for track one owns the underlying AMQP  connection manager and associated.   I think that we're likely to revisit that for the full track two implementation, but for this preview, I'd love to have a placeholder test marked `[Ignore]` with a description so that we don't forget the scenario later.

Reply:
Just added the Ignore Attribute. Should we have a similar scenario for Producer?

Reply:
Done for Producer.